src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -2659,7 +2659,7 @@ private EvmExceptionType InstructionLog<TTracing>(EvmState vmState, ref EvmStack
         where TTracingRefunds : struct, IIsTracing
         where TTracingStorage : struct, IIsTracing
     {
-        Metrics.SstoreOpcode++;
+        Metrics.IncrementSStoreOpcode();
```

Comment:
why do we need those Increment methods?

Reply:
To not expose the thread local outside metrics as its an implementation detail

Reply:
To follow up on this the property getter needs to be a vanilla `long` for graphana as it doesn't know what a `ThreadLocal<long>` is or how to process it; so when it's queried it sums all the thread local values to get a total value.

Can't `++` that value as its the sum of all threads; don't really want to expose the ThreadLocal directly as is implementation detail as it could be `Interlocked.Increment` or some other approach, etc

Reply:
Ok why not just add a setter that would set the ThreadLocal value - so the external code can just `++` on it?

Reply:
Ah it would read all threads and write to 1 thread. Would have to add `SstoreOpcodeLocal` that reads from threadlocal and increment that.